content/ProblemNajkrotszejSciezki.tex

```diff
@@ -499,34 +501,34 @@ \subsubsection{Algorytm A\texorpdfstring{$^*$}{TEXT}}
 	spełnia następujące założenia:
 	\begin{itemize}
 		\item dla każdego wierzchołka $v$ zachodzi $h(v) \leq \text{dist}(v, t)$,
-		\item dla każdej krawędzi $uv$ zachodzi $h(u) \leq h(v) + \omega(uv)$,
+		\item dla każdej krawędzi $uv$ zachodzi $h(u) \leq h(v) + w(uv)$,
 	\end{itemize}
 	to algorytm \ref{aStar_alg}
 	poprawnie rozwiązuje problem najkrótszej ścieżki
 	pomiędzy wierzchołkami $s$ i $t$.
 	\begin{proof}
-		(nie obowiązuje na kolokwium)
+		% \textbf{to do} (nie obowiązuje na kolokwium) % ;)

```

Comment:
Skoro nie obowiązuje do kolokwium to proponuję
_Originally posted by @kubazuch in https://github.com/migoox/aisd-notatki/pull/51#discussion_r1335221440_
```suggestion
		Dowód pozostawiamy czytelnikowi ;)
```